torch/optim/swa_utils.py

```diff
@@ -81,6 +88,8 @@ def ema_update(ema_param: Tensor, current_param: Tensor, num_averaged):
 
 
 def get_swa_avg_fn():
+    """Receive stochastic weight average."""
```

Comment:
and this one should be across a single param.

Reply:
Sure @janeyx99 Thanks so much for the quick review and suggestions! I will be updating the PR soon.